src/apollo-server/index.ts

```diff
@@ -145,12 +144,7 @@ export class OttofellerApolloServerProject extends TypeScriptAppProject {
     }
 
     // ANCHOR VSCode settings
-    VsCodeSettings.addToProject(this, options)
-
-    VsCodeSettings.of(this)?.add({
-      'eslint.useESLintClass': true,
-      'eslint.options': {cache: true, reportUnusedDisableDirectives: 'error'},
-    })
+    addVsCodeToProject(this)
```

Comment:
@quesabe let's follow the naming pattern (e.g. `addLinters` or `addHusky`):
```suggestion
    addVsCode(this)
```